aspnetcore-webapi/CustomHeaderParameterInSwagger/CustomHeaderParameterInSwagger/Controllers.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GlobalHeaderParameterInSwagger;
+
+[ApiController]
+[Route("api/[controller]")]
+public class SampleController : ControllerBase
+{
+    /// <summary>
```

Comment:
@lennart02 
Can we remove these too? Including the one in the next Action method